### PR TITLE
Metadata update for content view and content management

### PR DESCRIPTION
**Pull request by latran**
Add more metadata update from my component ownership for content view and content management.

### CODE REVIEW

File: tests/foreman/api/test_contentmanagement.py [hunk 1]
```diff
@@ -88,6 +89,8 @@ def test_positive_sync_repos_with_large_errata(self):
 
         :CaseLevel: Integration
 
+        :CaseImportance: High
```
Comment (ntkathole) [inline tests/foreman/api/test_contentmanagement.py]: We have it at https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/api/test_contentmanagement.py#L13. I guess no need to add it for every test then.

File: tests/foreman/api/test_contentviewfilter.py [hunk 2]
```diff
@@ -103,6 +107,8 @@ def test_positive_create_erratum_with_name(self):
             correct name and type
 
         :CaseLevel: Integration
+
+        :CaseImportance: High
```
Comment (ntkathole) [inline tests/foreman/api/test_contentviewfilter.py]: Same here, we have it above https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/api/test_contentviewfilter.py#L17

File: tests/foreman/api/test_contentviewversion.py [hunk 3]
```diff
@@ -160,6 +166,8 @@ def test_positive_promote_out_of_sequence_environment(self):
         :expectedresults: The promotion succeeds.
 
         :CaseLevel: Integration
+
+        :CaseImportance: High
```
Comment (ntkathole) [inline tests/foreman/api/test_contentviewversion.py]: Same for this file as well https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/api/test_contentviewversion.py#L13

**Comment by latran** [comment]
Removed CaseImportance: High redundancy based on suggestions.

**Pull request merged**